addons/mass_mailing/views/mass_mailing_views.xml

```diff
@@ -12,20 +12,70 @@
             parent="mass_mailing_menu_root" sequence="100"/>
 
         <!--  MAILING CONTACT -->
+        <record model="ir.ui.view" id="mail_mass_mailing_list_contact_rel_form">
+            <field name="name">mail.mass_mailing.list_contact_rel.form</field>
+            <field name="model">mail.mass_mailing.list_contact_rel</field>
+            <field name="priority">10</field>
+            <field name="arch" type="xml">
+                <form string="Mailing List contacts">
+                    <sheet>
+                        <group>
+                            <field name="list_id"/>
+                            <field name="is_blacklisted" invisible="1"/>
+                            <label for="contact_id" class="oe_inline"/>
+                            <div class="o_row o_row_readonly">
+                                <i class="fa fa-ban" style="color: red;" role="img" title="This email is blacklisted for mass mailing"
+                                    aria-label="Blacklisted" attrs="{'invisible': [('is_blacklisted', '=', False)]}"></i>
+                                <field name="contact_id"/>
+                            </div>
+                            <field name="creation_date" readonly="1"/>
+                            <field name="unsubscription_date" readonly="1"/>
+                            <field name="opt_out"/>
+                            <field name="message_bounce" readonly="1"/>
+                        </group>
+                    </sheet>
+                </form>
+            </field>
+        </record>
 
-        <!-- Custom reports (aka filters) -->
-        <record id="filter_contact_subscription" model="ir.filters">
-            <field name="name">List Subscription</field>
-            <field name="model_id">mail.mass_mailing.contact</field>
-            <field name="user_id" eval="False"/>
-            <field name="context">{'group_by': ['create_date:month']}</field>
+        <record model="ir.ui.view" id="mail_mass_mailing_list_contact_rel_list_contact_tree">
+            <field name="name">mail.mass_mailing.list_contact_rel.list_contact.tree</field>
```

Comment:
Guidelines: don't forget the .view. (.view.form/tree). Otherwise you make @jem-odoo very sad.